sdk/core/azure-core/src/main/java/com/azure/core/util/tracing/StartSpanOptions.java

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.core.util.tracing;
+
+import com.azure.core.annotation.Fluent;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Represents span options that are available before span starts and describe it.
+ */
+@Fluent
+public class StartSpanOptions {
+
+    /**
+     * Type of Span. Only CLIENT and INTERNAL kinds are supported.
+     */
+    public enum Kind {
```

Comment:
Why we creating new enum Kind , and not using Otel SpanKind ? Is this due to the fact we support only two ?

Reply:
This is because azure-core can't depend on the otel, but cosmos and other SDKs have to use core. I added only two because there is no need for other kinds just yet.